ko/documentation/ruby-from-other-languages/to-ruby-from-perl/index.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: page
+title: "Perl에서 Ruby로"
+lang: ko
+---
+
+펄은 굉장합니다. Perl의 문서도 굉장합니다. Perl 커뮤니티도… 굉장합니다.
+처음부터 내장된 우아한 OO 기능을 갈망하는 Perlers에게는 Ruby가 적합할 수
```

Comment:
```suggestion
처음부터 내장된 우아한 XX 기능을 갈망하는 Perlers에게는 Ruby가 적합할 수
```
(maybe) better to use 'xx' for korean? 🤔

Reply:
I thought it stand for Object Oriented

Reply:
Japanese say
> 最初から洗練されたオブジェクト指向の機能が組み込まれた言語を求めているPerl使いにとって、 Rubyは答えになるかもしれません。

So, It is Object Oriented. maybe We can replace it like:
```suggestion
처음부터 내장된 우아한 객체 지향 기능을 갈망하는 Perlers에게는 Ruby가 적합할 수
```

Reply:
oh 🙈 lgtm